app/src/main/java/net/gsantner/markor/util/AppSettings.java

```diff
@@ -371,15 +379,30 @@ public boolean getDocumentWrapState(final String path) {
 
     public void setDocumentFormat(final String path, final int format) {
```

Comment:
These routines have been updated to store and retrieve format by _index_. This is now robust to the res value changing on recompilation